share/generate_date_holidays_gb.pl

```diff
@@ -10,7 +10,7 @@
 use DateTime;
 use File::Spec::Functions qw( catfile splitpath updir );
 use JSON qw(decode_json);
-use List::Util qw( uniq );
+use List::Util 1.45 qw( uniq );
```

Comment:
FWIW, List::MoreUtils has a uniq which seemed to work fine and has been around longer.